doc/uftrace-live.md

```diff
@@ -355,6 +355,28 @@ In addition, you can enable all functions at load time using '.' that matches to
        2.451 us [19387] |   } /* a */
        3.289 us [19387] | } /* main */
 
+Clang/LLVM 4.0 provides a dynamic tracing feature called [X-ray](http://llvm.org/docs/XRay.html).  It's similar to `gcc -mfentry -mnop-mcount` and `-finstrument-functions`.  The uftrace tool also supports `X-ray`.
```

Comment:
s/dynamic tracing feature/dynamic instrumentation technique/
s/similar to/similar to a combination of/

And the last sentence can be rewritten as:

The uftrace also supports dynamic tracing on the executables built with the `X-ray`.